### ISSUE TITLE
fatal error: 'longintrepr.h' file not found

### ISSUE THREAD

**Issue by henrydatei** [fabanc/pyvoronoi#21]
When installing with pip I get the above error:
```
Collecting pyvoronoi
  Using cached pyvoronoi-1.0.7.tar.gz (147 kB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: pyvoronoi
  Building wheel for pyvoronoi (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      Distribution mode: Compiling Cython generated .cpp sources.
      running bdist_wheel
      running build
      running build_ext
      building 'pyvoronoi' extension
      creating build
      creating build/temp.macosx-12-x86_64-cpython-311
      creating build/temp.macosx-12-x86_64-cpython-311/pyvoronoi
      clang -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -Ipyvoronoi -I/usr/local/opt/python@3.11/Frameworks/Python.framework/Versions/3.11/include/python3.11 -c pyvoronoi/pyvoronoi.cpp -o build/temp.macosx-12-x86_64-cpython-311/pyvoronoi/pyvoronoi.o
      pyvoronoi/pyvoronoi.cpp:198:12: fatal error: 'longintrepr.h' file not found
        #include "longintrepr.h"
                 ^~~~~~~~~~~~~~~
      1 error generated.
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyvoronoi
  Running setup.py clean for pyvoronoi
Failed to build pyvoronoi
Installing collected packages: pyvoronoi
  Running setup.py install for pyvoronoi ... error
  error: subprocess-exited-with-error
  
  × Running setup.py install for pyvoronoi did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      Distribution mode: Compiling Cython generated .cpp sources.
      running install
      /usr/local/lib/python3.11/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_ext
      building 'pyvoronoi' extension
      creating build
      creating build/temp.macosx-12-x86_64-cpython-311
      creating build/temp.macosx-12-x86_64-cpython-311/pyvoronoi
      clang -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -Ipyvoronoi -I/usr/local/opt/python@3.11/Frameworks/Python.framework/Versions/3.11/include/python3.11 -c pyvoronoi/pyvoronoi.cpp -o build/temp.macosx-12-x86_64-cpython-311/pyvoronoi/pyvoronoi.o
      pyvoronoi/pyvoronoi.cpp:198:12: fatal error: 'longintrepr.h' file not found
        #include "longintrepr.h"
                 ^~~~~~~~~~~~~~~
      1 error generated.
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> pyvoronoi

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
```
This error seems to occur with Python 3.11 (some other packages had this problem too after Python 3.11 came out)

**Comment by Nemoumbra** [comment]
Quite true... This makes using the py-pfd-parser's visualizer impossible.
This is how aiohttp dealt with the issue: https://github.com/aio-libs/aiohttp/issues/6600?ysclid=ll8jsxipxn389328965

**Comment by fabanc** [comment]
Might be onto a fix for that. See issue #22. I need users who can test the fix on Mac, as I "only" have access to Windows and Ubuntu.

**Comment by fabanc** [comment]
Issue solved with commit #25 e1ae424. I have created a task so that the pypi project gets updated when I publish on GitHub, so `pip install pyvoronoi` will get you the latest version.

I have tested on Ubuntu from 3.9 to 3.12 using Docker. I have had a few Windows image testing 3.9, 3.11, 3.12. The only thing I could not test on is Mac OS. 

@h0h0h0 , I think I granted your wish. Go ahead and have fun! Note that the URL of the repository has changed since @Voxel8 transferred the ownership of that repo to me. It is good to keep that in mind.
